tests/instructorAssessmentDownloads.test.js

```diff
@@ -2,7 +2,8 @@ const _ = require('lodash');
 const assert = require('chai').assert;
 const request = require('request');
 const cheerio = require('cheerio');
-const csvParse = require('csv-parse/lib/sync');
+// eslint-disable-next-line import/no-unresolved -- https://github.com/import-js/eslint-plugin-import/issues/2132
+const { parse: csvParse } = require('csv-parse/sync');
```

Comment:
https://github.com/import-js/eslint-plugin-import/issues/1810 - they're blocked on `resolve` supporting `exports` in `package.json`.


```suggestion
// eslint-disable-next-line import/no-unresolved -- https://github.com/import-js/eslint-plugin-import/issues/1810
const { parse: csvParse } = require('csv-parse/sync');
```